Content.Client/Access/UI/IdCardConsoleBoundUserInterface.cs

```diff
@@ -23,16 +24,15 @@ public IdCardConsoleBoundUserInterface(EntityUid owner, Enum uiKey) : base(owner
         protected override void Open()
         {
             base.Open();
-            List<ProtoId<AccessLevelPrototype>> accessLevels;
+            List<string> accessLevels;
```

Comment:
Seems like this is a revert. Could you adapt the other code to use `List<ProtoId<AccessLevelPrototype>>` instead of reverting?

Reply:
There wasn't `ProtoId` when I created this PR. Maybe I somehow reverted it on merge.
But it doesn't matter, it's all ProtoId now.